File: HomeAutomation/HomeAutomation/src/EnvironmentState.java
public class EnvironmentState {
    public double temperature;
    public boolean motionDetected;
    public double lightLevel;

    public EnvironmentState() {}

    public EnvironmentState(double temperature, boolean motionDetected, double lightLevel) {
        this.temperature = temperature;
        this.motionDetected = motionDetected;
        this.lightLevel = lightLevel;
    }

    @Override
    public String toString() {
        return "EnvironmentState{" +
                "temperature=" + temperature +
                ", motionDetected=" + motionDetected +
                ", lightLevel=" + lightLevel +
                '}';
    }
}

File: HomeAutomation/HomeAutomation/Core/EnvironmentState.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HomeAutomation.Core
{
    public class EnvironmentState
    {
        public bool MotionDetected { get; set; }
        public int AmbientLight { get; set; }
        public double Temperature { get; set; }
        public double PowerUsage { get; set; }
    }

}
